src/code/audio/audio_dcache.c

```diff
@@ -0,0 +1,15 @@
+#include "global.h"
+
+void Audio_InvalDCache(void* buf, s32 size) {
+    OSIntMask prevMask = osSetIntMask(1);
+
+    osInvalDCache(buf, size);
+    osSetIntMask(prevMask);
+}
+
+void Audio_WritebackDCache(void* buf, s32 size) {
```

Comment:
Hmm so this is a cache size. Thinking size_t is the appropriate type here. Anyone else have any input?

```suggestion
void Audio_WritebackDCache(void* buf, size_t size) {
```

Reply:
Agreed, it's been changed for both functions. It's also being changed in an open PR for the OoT equivalent.